apps/hip/static/js/sidebar.js

```diff
@@ -16,6 +16,25 @@ export default function () {
       }
     }
 
+    function getParentSidebarLink (currentPath) {
+      /* Use Breadcrumb navigation to determine which SideBarLink is active
+
+      This function's logic is based on the assumption that sections that
+      have a page depth of greater than 1 will use breadcrumb navigation.
```

Comment:
Another assumption that could be added is that the first `<a>` of the breadcrumb will have an href that matches one of the sidebar `data-href` attributes

Reply:
thanks and done.